_15thnight/__init__.py

```diff
@@ -1,5 +1,6 @@
 """15th Night Flask App."""
 
+from celery import Celery
 from email_client import send_email, verify_email
```

Comment:
I think this is the line that should be removed, not the `send_email` import removed below (leave that one).